### PR TITLE
Created artifactory script

### PR DESCRIPTION
**Pull request by andrii-prism**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -144,6 +144,10 @@ function(buildSdk)
         tar -czf ${PROJECT_SOURCE_DIR}/${PROJECT_NAME}_${SDK_VER}_${REV_HASH}_${PRISM_PLATFORM}.${CONNECT_BUILD_TYPE}.tar.gz ${DELIVERY_FILES}
         WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX})
 
+    add_custom_target(artifactory
```
Comment (Yurmusp) [inline CMakeLists.txt]: I would call it upload_delivery or artifactory_delivery instead of artifactory.

File: build.sh [hunk 2]
```diff
@@ -23,6 +24,8 @@ parse_cmd_line(){
         BUILD_TYPE="${i#*=}";;
         --delivery)
         MAKE_DELIVERY=1;;
+        --upload-delivery)
```
Comment (dbus-prsm) [inline build.sh:28]: Add new argument description into the script header.

File: CMakeLists.txt [hunk 3]
```diff
@@ -144,6 +144,10 @@ function(buildSdk)
...
+        jfrog rt u ${PROJECT_SOURCE_DIR}/${PROJECT_NAME}_${SDK_VER}_${REV_HASH}_${PRISM_PLATFORM}.${CONNECT_BUILD_TYPE}.tar.gz libs-release-local/${PRISM_PLATFORM}/
```
Comment (dbus-prsm) [inline CMakeLists.txt]: You should use a variable to store the archive file name in order to avoid a discrepancy in naming for delivery targets.

**Review by dbus-prsm — CHANGES_REQUESTED** [review]

File: build.sh (hunk 2 above)
Comment (andrii-prism) [reply build.sh:28]: done

**Review by dbus-prsm — APPROVED** [review]

File: CMakeLists.txt [hunk 4]
```diff
@@ -140,8 +140,14 @@ function(buildSdk)
...
     message("git revision short hash: ${REV_HASH}")
 
+    set(DELIVERY_NAME ${PROJECT_SOURCE_DIR}/${PROJECT_NAME}_${SDK_VER}_${REV_HASH}_${PRISM_PLATFORM}.${CONNECT_BUILD_TYPE}.tar.gz)
+
     add_custom_target(delivery
-        tar -czf ${PROJECT_SOURCE_DIR}/${PROJECT_NAME}_${SDK_VER}_${REV_HASH}_${PRISM_PLATFORM}.${CONNECT_BUILD_TYPE}.tar.gz ${DELIVERY_FILES}
+        tar -czf ${DELIVERY_NAME}  ${DELIVERY_FILES}
+        WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX})
+
+    add_custom_target(upload_artifactory
```
Comment (Yurmusp) [inline CMakeLists.txt:149]: either upload_artifact or upload_to_artifactory

**Pull request merged**